GliaWidgets/Coordinator/RootCoordinator.swift

```diff
@@ -36,7 +36,11 @@ class RootCoordinator: SubFlowCoordinator, FlowCoordinator {
     private let isWindowVisible = ObservableValue<Bool>(with: false)
     private let navigationController = NavigationController()
     private let navigationPresenter: NavigationPresenter
-    private var window: GliaWindow?
+    private let presentingWindow = UIApplication.shared.windows.filter { $0.isKeyWindow }.first
+    private var presentingViewController: UIViewController? {
+        return presentingWindow?.topMostViewController()
+    }
+    private var gliaViewController: GliaViewController!
```

Comment:
Using implicitly unwrapped optional could make a side effect and emits crash, so I'd like to propose to avoid using it (if possible). Probably we could use optional or lazy variable instead. Does it make sense?